src/main/java/com/zufar/onlinestore/payment/enums/PaymentStatus.java

```diff
@@ -1,12 +1,38 @@
 package com.zufar.onlinestore.payment.enums;
 
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
 public enum PaymentStatus {
-    REQUIRES_PAYMENT_METHOD,
-    REQUIRES_CONFIRMATION,
-    REQUIRES_CAPTURE,
-    REQUIRES_ACTION,
-    PROCESSING,
-    CANCELED,
-    SUCCEEDED
+
+    REQUIRES_PAYMENT_METHOD("payment_intent.requires_payment_method",
+            "Payment method is required"),
+
+    REQUIRES_CONFIRMATION("payment_intent.requires_confirmation",
```

Comment:
Look better in the context of English
`CONFIRMATION_IS_REQUIRED("payment_intent.confirmation_is_required"`